README.md

```diff
@@ -111,16 +111,42 @@ Hello World!
 
 Stargz snapshotter also supports [further configuration](/docs/overview.md) including private registry authentication, mirror registries, etc.
 
-## Creating eStargz images with optimization
+## Getting eStargz images
 
 - For more examples and details about the image converter `ctr-remote`, refer to [Optimize Images with `ctr-remote image optimize`](/docs/ctr-remote.md).
 - For more details about eStargz format, refer to [eStargz: Standard-Compatible Extensions to Tar.gz Layers for Lazy Pulling Container Images](/docs/stargz-estargz.md)
 
 For lazy pulling images, you need to prepare eStargz images first.
-You can use [`ctr-remote`](/docs/ctr-remote.md) command for do this.
-You can also try our pre-converted images listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+There are several ways to achieve that.
+This section describes some of them.
 
-In this section, we introduce `ctr-remote` command for converting images into eStargz with optimization for reading files.
+### Trying pre-built eStargz images
+
+You can try our pre-converted eStargz images on ghcr.io listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+
+### Registry-side conversion with `estargz.kontain.me`
```

Comment:
Can you include some warning about stability/security of relying on my weekend side project? It's probably best used to easily try it out on whatever image you want, but if you start to rely on it in production you should either copy images to your own real registry, or build them yourself with `ctr-remote` as described below.

Reply:
Thank you for the suggestion. added an warning message to README. LGTY?

Reply:
Looks great! 👍